src/components/issues/details/Label.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { withRouter } from 'react-router-dom';
+import { addLabel, deleteLabel } from 'actions/issues.actions';
+
+import labelsSelector from 'selectors/labels.selector';
+
+import check from 'assets/images/check.svg';
+import exit from 'assets/images/exit.svg';
+
+type OwnProps = {
+  match: Object
+};
+
+type ConnectedProps = {
+  deleteLabel: () => void,
+  addLabel: () => void
+};
+
+class Label extends Component<OwnProps & ConnectedProps> {
+  handleDeActiveLabel = ({ name }) => {
+    this.props.deleteLabel(this.props.match.params, name);
+  };
+
+  handleActiveLabel = label => {
+    if ('default' in label) {
```

Comment:
I think it would be a better practice to use get and not only rely on the existence of a property. it would take into account falsey values and prevent side effects (not in this case specifically). And on another note it would be better to be consistent across the application. some places use 'in' operator and some hasIn. please take into account